apps/web/src/components/profile/my-profile-email.tsx

```diff
@@ -38,7 +38,7 @@ export default function MyProfileEmail() {
         password: formData.get('password') as string,
       }
       const bodyValidation = await validate(body)
-      if (bodyValidation.isValid) {
+      if (bodyValidation.state === 'valid') {
```

Comment:
`isValid` is deprecated due to a broken TS typing. Comparing `state` against `'valid'` and `'invalid'` is preferred.